iOSiTunesSearch/iOSiTunesSearch/Controllers/SearchResultTableViewCell.swift

```diff
@@ -0,0 +1,9 @@
+import UIKit
+
+class SearchResultTableViewCell: UITableViewCell {
+    static let reuseIdentifier = "cell"
```

Comment:
Next time, try to make a reuse identifier that is indicative of the kind of cell that the identifier represents. That way if you have more than one cell, your identifiers won't conflict and/or confuse you.